meson.build

```diff
@@ -53,6 +71,15 @@ foreach s : schemas
   )
 endforeach
 
+# Compile schemas manually
+glib_compile = find_program('glib-compile-schemas', required: true)
+
+run_command([glib_compile, 'schemas'])
+install_data('schemas/gschemas.compiled', install_dir: schemadir)
+
+run_command([glib_compile, 'migration'])
+install_subdir('migration', install_dir: join_paths(extensiondir, uuid))
```

Comment:
I would probably put this under schemas (as I did in the panel extension) but detail...

Reply:
I opted to move it to a different folder mostly because this schema is trated completely differently than the one under `schemas/`. It's installed in a different folder, read differently, etc.

Reply:
That is fine, in the panel extension I just install it in the same folder and then only have to compile the schemas once, but that is a detail - can change there also if you prefer... for reference this is on https://github.com/endlessm/eos-panel-extension/pull/7.